### PR TITLE
Fixed schedule generation for even number of teams ref #35

### PR DESCRIPTION
**Pull request by Askir**
Just properly implementing round robin :)

### CODE REVIEW

**Comment by Askir** [comment]
Could you have a look at this @erksch ? I'm pretty sure it works but maybe check it out yourself and try with a few team sizes. The current version only works for uneven teams :sweat_smile:

File: app/models/event.rb [hunk 1]
```diff
@@ -61,30 +61,29 @@ def add_test_teams
   end
 
   def generate_schedule
-    calculate_gamedays
+    calculate_round_robin
   end
 
   def gamedays
```
Comment (erksch) [inline app/models/event.rb]: Do you still need this function? Seems like your not using it anymore.

**Comment by erksch** [comment]
What about the uneven numbers of teams?

**Comment by erksch** [comment]
@erksch okay nevermind

**Review by erksch — APPROVED** [review]

**Pull request merged**